README.md

```diff
@@ -8,3 +8,23 @@
 * It uses the [GCI private API](https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf)
 * __Attention:__ This repository only re-publishes publicly available information
 * Uses IGitt, supporting both GitHub & GitLab
+
+## Deploy Netlify on your fork :
+
+Deploy netlify on your forked ``community`` repository.
+After signup on [netlify](https://app.netlify.com/).
+
+- Click on **New site from Git**.
+- Choose **Git provider (Github)** > Choose a repository **< username >/community**.
+- Branch to deploy **master**.
+- Basic build settings :
+  * Set Build command to **.ci/build.sh**.
+  * Set Publish directory to **public**.
+  * Click Deploy site to deploy the site.
```

Comment:
No need to click on the deploy button before setting the environment variables. There is a button name `Show advanced` after clicking on it you can add all of your environment variables, keys and values there.(e.g: in our case we need to put two keys: 1. GH_TOKEN and 2.OH_TOKEN) and then click on `Deploy site`.